sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMiningAsync.cs

```diff
@@ -33,33 +34,33 @@ public async Task AnalyzeSentimentWithOpinionMiningAsync()
                 "We changed rooms as the toilet smelled."
             };
 
-            AnalyzeSentimentResultCollection reviews = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { AdditionalSentimentAnalyses = AdditionalSentimentAnalyses.OpinionMining });
+            AnalyzeSentimentResultCollection analyzedReviews = await client.AnalyzeSentimentBatchAsync(reviews, options: new AnalyzeSentimentOptions() { MineOpinions = true });
 
-            Dictionary<string, int> complaints = GetComplaint(reviews);
+            Dictionary<string, int> complaintTargets = GetNegativeOpinionTargets(analyzedReviews);
 
-            var negativeAspect = complaints.Aggregate((l, r) => l.Value > r.Value ? l : r).Key;
-            Console.WriteLine($"Alert! major complaint is *{negativeAspect}*");
+            var negativeOpinionTarget = complaintTargets.Aggregate((a, b) => a.Value > b.Value ? a : b).Key;
```

Comment:
I changed l, r to a, b b/c it's more conventional for arbitrary variable names.

Reply:
didn't know that. thanks!